http-clients/src/main/java/com/palantir/remoting/http/FeignClients.java

```diff
@@ -41,8 +43,11 @@ private FeignClients() {}
     public static FeignClientFactory standard() {
         return FeignClientFactory.of(
                 new GuavaOptionalAwareContract(new JAXRSContract()),
-                new JacksonEncoder(ObjectMappers.guavaJdk7()),
-                new OptionalAwareDecoder(new TextDelegateDecoder(new JacksonDecoder(ObjectMappers.guavaJdk7()))),
+                new InputStreamDelegateEncoder(new JacksonEncoder(ObjectMappers.guavaJdk7())),
+                new OptionalAwareDecoder(
```

Comment:
what's the alternative? if it finds an InputStream it processes, otherwise it delegates (this matches the rest of our stuff?)